contents/docs/settings/access-control.mdx

```diff
@@ -65,48 +67,95 @@ You can override the default access level for specific members or [roles](#role-
     classes="rounded"
 />
 
+See the table below for a summary of project-level permissions:
+
+| Permission                                                                        | Member                                           | Admin                                            |
+|-----------------------------------------------------------------------------------|--------------------------------------------------|--------------------------------------------------|
+| Manage project access controls                                                    | <span className="text-red text-lg">✖</span>      | <span className="text-green text-lg">✔</span>    |
+| Delete project                                                                    | <span className="text-red text-lg">✖</span>      | <span className="text-green text-lg">✔</span>    |
+| Edit project settings                                                             | <span className="text-red text-lg">✖</span>      | <span className="text-green text-lg">✔</span>    |
+| View/edit own or permitted resources (based on resource-level access controls)    | <span className="text-green text-lg">✔</span>    | <span className="text-green text-lg">✔</span>    |
+| View/edit all resources (regardless of resource-level access controls)            | <span className="text-red text-lg">✖</span>      | <span className="text-green text-lg">✔</span>    |
+
 ### 3. Resource level
 
-> This is currently being rolled out to all users so you may not see this experience yet. If you're interested in trying it out please email zach@posthog.com with "Access control beta" in the subject line.
+> This feature is currently being rolled out to all users, so you may not see this experience yet. If you're interested in trying it out, please email zach@posthog.com with "Access control beta" in the subject line.
+
+Resource access controls allow you to control who can view and edit specific resource objects. These can be accessed in the "Access control" sidebar when viewing a supported resource.
 
-Access control is available for:
+Currently, resource access controls are available for:
 - Insights
 - Dashboards
 - Notebooks
 - Feature flags
-- (more resource types coming soon)
+- (more resource types coming soon – looking for others? Let us know!)
 
-Each resource has three possible access levels:
-- **No access** - Cannot view or edit the resource
-- **View** - Can view but not modify the resource
-- **Edit** - Can view and modify the resource
+Note: We do not yet support limiting access to querying data, viewing replays, or accessing people data. Support for these features is planned for the near future.
```

Comment:
"people data" part sounds a bit vague, maybe "person profiles" for consistency with naming elsewhere?

Reply:
Good point - will update.